
epoch: 0 => loss = 0.42220738114753215
epoch: 1 => loss = 0.3740480203225132
Traceback (most recent call last):
  File "main.py", line 70, in <module>
    test_predict = nn.feedforward(X_test)
  File "/home/tharikh/dl-ass-1/network.py", line 90, in feedforward
    a = self.W[0].T @ X + self.b[0]
ValueError: matmul: Input operand 1 has a mismatch in its core dimension 0, with gufunc signature (n?,k),(k,m?)->(n?,m?) (size 28 is different from 784)